src/timestamp.ts

```diff
@@ -0,0 +1,5 @@
+export let timestamp = 1;
```

Comment:
We could start from `Number.MIN_SAFE_INTEGER`? Also, remove `export` as pointed out by @Jomik.

Reply:
the timestamp could be useful for debugging, meaning that starting from 0 and up would be better.
I believe we still have plenty of numbers left. :smile:

Reply:
Good point. Then what about initializing to `-1` so that the first timestamp is `0`?